backend/lib/routes/info.js

```diff
@@ -58,3 +61,33 @@ async function fetchGardenerVersion () {
     throw err
   }
 }
+
+router.route('/openapi/shoot')
```

Comment:
move one level up.

Reply:
I propose to have a route a route `openapi` or `schemas`  which returns all schemas required by the frontend. In our case:
```json
{ 
  "definitions": {
   "com.github.gardener.gardener.pkg.apis.core.v1alpha1.Shoot": {
      ...
    }
  }
}
```